resources/bin/bal.bat

```diff
@@ -81,15 +81,14 @@ if "%RUN_BALLERINA%" == "true" (
     for /f %%a in (%CURRENT_PATH%\..\distributions\ballerina-version) do (
         set BALLERINA_HOME=%%a
     )
-    if exist %userprofile%\.ballerina\ballerina-version (
+    if exist "%userprofile%\.ballerina\ballerina-version" (
         set "FILE_PATH=%userprofile%\.ballerina\ballerina-version"
     )
 
     SetLocal EnableDelayedExpansion
-    for /f %%a in (!FILE_PATH!) do (
-        if exist %CURRENT_PATH%..\distributions\%%a (
-            set BALLERINA_HOME=%%a
-        )
+    set /p CURRENT_VERSION=< "!FILE_PATH!"
```

Comment:
```suggestion
    set /p CURRENT_VERSION =< "!FILE_PATH!"
```

Reply:
@pramodya1994 
Addressed the suggestions